src/Network/Network.Test/ScenarioTests/ExpressRoutePortsMacSecTest.ps1

```diff
@@ -0,0 +1,154 @@
+﻿# ----------------------------------------------------------------------------------
```

Comment:
this test case need to be recorded

Reply:
[Added  record](https://github.com/Azure/azure-powershell/commit/68de3c7b2300f1ecbafb2b7aecc668a7f6471994)